telehealth-example/src/main/java/com/pubnub/components/example/getting_started/ui/view/Login.kt

```diff
@@ -0,0 +1,209 @@
+package com.pubnub.components.example.getting_started.ui.view
+
+import android.content.Context
+import android.content.Intent
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.ContextCompat.startActivity
+import com.pubnub.components.data.message.asMap
+import com.pubnub.components.example.getting_started.ChannelActivity
+import com.pubnub.components.example.getting_started.DefaultDataRepository
+import com.pubnub.components.example.getting_started.R
+
+object Login {
```

Comment:
View is not scrollable. Doesn't look good on small devices:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/7523141/197472571-9a275e1b-9108-4c42-b123-b539c799cc92.png">

Reply:
Additionally, the link navigates to `google.com`

Reply:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/7523141/197473921-2231e83f-62ee-4cff-b4af-111cfc1f6d3a.png">

Reply:
Done